app/controllers/api/v1/items_controller.rb

```diff
@@ -0,0 +1,41 @@
+class Api::V1::ItemsController < ApplicationController
+  def index
+    @items = Item.all
+    render json: @items
+  end
+
+  def show
+    item_id = params[:id]
+    @item = Item.find(item_id)
+    render json: @item
+  end
+
+  def create
+    @image = Cloudinary::Uploader.upload(params[:image])
```

Comment:
Let's also include image as part of our sanitized parameters (item_params). A malicious person can send a POST request with scripts that can download data (SQLi attack) if we don't sanitize all of our user inputs